utils/chat.js

```diff
@@ -36,8 +39,13 @@ const generateReplyDataFromMessage = (message, userData, isAnonimous = true) =>
     };
   }
 
+  const {id, type, created_at, updated_at} = message || {};
+
   return {
-    ...message,
+    id,
+    type,
+    created_at,
+    updated_at,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code is extracting only `id`, `type`, `created_at`, and `updated_at` from the `message` object. This could potentially lead to data loss if there are other important properties in the `message` object that are not being included in the returned object. Please ensure that all necessary properties are being extracted or consider reverting back to spreading the entire `message` object.



<!-- This is an auto-generated comment by OSS CodeRabbit -->